x-element.js

```diff
@@ -1,15 +1,648 @@
-/**
- * Base class for custom elements.
- */
-
-import ElementMixin from './mixins/element-mixin.js';
-import ListenersMixin from './mixins/listeners-mixin.js';
-import LitHtmlMixin from './mixins/lit-html-mixin.js';
-import PropertiesMixin from './mixins/properties-mixin.js';
-import PropertyEffectsMixin from './mixins/property-effects-mixin.js';
-
-export default LitHtmlMixin(
-  ListenersMixin(
-    PropertyEffectsMixin(PropertiesMixin(ElementMixin(HTMLElement)))
-  )
-);
+import { render, html, svg, directive } from '../../lit-html/lit-html.js';
+import { asyncAppend } from '../../lit-html/directives/async-append.js';
+import { asyncReplace } from '../../lit-html/directives/async-replace.js';
+import { cache } from '../../lit-html/directives/cache.js';
+import { guard } from '../../lit-html/directives/guard.js';
+import { ifDefined } from '../../lit-html/directives/if-defined.js';
+import { repeat } from '../../lit-html/directives/repeat.js';
+import { unsafeHTML } from '../../lit-html/directives/unsafe-html.js';
```

Comment:
is `unsafeSVG` overkill? any other directives left out here?

Reply:
also — `classMap`, `live`, `style-map`, `template-content`

Reply:
can we get `live` in, it's useful for controlled components like `input`

Reply:
Updated `lit-html` to get all the new goodies and added them all. Note that we should probably make a quick PR to master that updates `lit-html` to ensure interop before merging and using this one now.